common/src/main/resources/assets/ftbchunks/lang/fr_fr.json

```diff
@@ -0,0 +1,76 @@
+{
+	"ftbchunks": "FTB Chunks",
+	"ftbchunks.noise": "Brouillage",
+	"ftbchunks.shadows": "Ombres",
+	"ftbchunks.chunk_grid": "Quadrillage",
+	"ftbchunks.reduced_color_palette": "Plaette de couleurs réduite",
+	"ftbchunks.saturation": "Saturation",
+	"ftbchunks.debug_info": "Infos de débug",
+	"ftbchunks.claimed_chunks_on_map": "Montrer les chunks revendiqués sur la carte",
```

Comment:
```suggestion
	"ftbchunks.claimed_chunks_on_map": "Montrer les tronçons revendiqués sur la carte",
```

Reply:
"Chunk" feels like a word that is sufficiently used within the french Minecraft community to be used as such. Even the french fandom page says that it is "more commonly called a _chunk_", as it says [here](https://minecraft.fandom.com/fr/wiki/Tron%C3%A7on).
I would even argue that using "tronçon" could be confusing and inconsistent with a mod that has "chunks" in its name.
For these reasons I will resolve this conversation without merging these suggestions. Thank you very much for them (and the others) though @ThalusA!